modules/win_disable_services/manifests/disable_uac.pp

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class win_disable_services::disable_uac {
+  case $facts['custom_win_os_version'] {
+    'win_11_2009':{
+      registry_value { 'HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\EnableLUA':
+        type => dword,
+        data => '0',
+      }
+    }
+    default: {
+      fail("${module_name} does not support ${$facts['custom_win_os_version']}")
```

Comment:
Let's change this from `fail` to `warning`. That way it won't break images builds for existing pools. Though in the long term we should add this to all pools.